packages/frontend/src/hooks/echarts/useEchartsCartesianConfig.ts

```diff
@@ -1860,6 +1863,24 @@ const useEchartsCartesianConfig = (
                     })
                     .join('');
 
+                // At the moment, we only correctly filter fields that are
```

Comment:
Check comment, we only can rely on the data from the chart, which is already prefiltered. If we want to use other fields, we'll have to rely on resultsData, and I believe that will impact performance. 

We can release this version first, and then wait for feedback (I want to see other use cases)